extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1040,9 +1040,9 @@ export class AmpStoryPage extends AMP.BaseElement {
    * @private
    */
   preloadAllMedia_() {
-    return this.whenAllMediaElements_((mediaPool, mediaEl) => {
-      this.preloadMedia_(mediaPool, mediaEl);
-    });
+    return this.whenAllMediaElements_((mediaPool, mediaEl) =>
+      this.preloadMedia_(mediaPool, mediaEl)
+    );
```

Comment:
Before it was not waiting for the promise, now with a lambda function it returns the promise so it waits for it.